packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -148,6 +147,8 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
             }
             key={ selectedIndex }
             aria-atomic={ true }
+            role='textbox'
+            aria-readonly='true'
```

Comment:
Aria guidelines for Combobox state:

The first element within the combobox is an input text field and is responsible for managing the keyboard focus between the text field and the list as well as displaying the list. The text field is in the tab order. If you create a text field without using a standard HTML text field form control then ensure that it is in the tab order.

If the text field is not editable it must have have aria-readonly = true.